api/interestr/api/views.py

```diff
@@ -31,7 +31,7 @@
 
 ### List Views BEGIN
 
-class UserList(generics.ListAPIView):
+class UserList(generics.ListCreateAPIView):
```

Comment:
This allows an undesired creation of a user without password from the API.

Reply:
My bad. Correcting immediately.